tests/libmigration/migration.go

```diff
@@ -525,7 +525,7 @@ func RunMigrationAndCollectMigrationMetrics(vmi *v1.VirtualMachineInstance, migr
 	getKubevirtVMMetricsFunc := tests.GetKubevirtVMMetricsFunc(&virtClient, pod)
 	Eventually(func() error {
 		out := getKubevirtVMMetricsFunc(ip)
-		lines := libinfra.TakeMetricsWithPrefix(out, "kubevirt_migrate_vmi")
+		lines := libinfra.TakeMetricsWithPrefix(out, "kubevirt_vmi_migration")
```

Comment:
@iholder101, @enp0s3 IIUC, this test checks if the following group of metrics (and their new names) has been created:
```
|kubevirt_migrate_vmi_data_processed_bytes | kubevirt_vmi_migration_data_processed_bytes|
|kubevirt_migrate_vmi_data_remaining_bytes | kubevirt_vmi_migration_data_remaining_bytes|
|kubevirt_migrate_vmi_dirty_memory_rate_bytes | kubevirt_vmi_migration_dirty_memory_rate_bytes|
|kubevirt_migrate_vmi_disk_transfer_rate_bytes | kubevirt_vmi_migration_disk_transfer_rate_bytes|
|kubevirt_migrate_vmi_memory_transfer_rate_bytes | kubevirt_vmi_migration_memory_transfer_rate_bytes|
```
Can we count on `TakeMetricsWithPrefix()` to take only this group of metric after its prefix input has changed to the common migration metrics prefix? If not, we may have a bug in the test, in which this group wasn't created, but the test passes since with this PR we have other metrics with this prefix.

Reply:
So this should be changed to "kubevirt_vmi_migration_" right?

Reply:
Still, we have `kubevirt_vmi_migration_failed` and `kubevirt_vmi_migration_succeeded` with this prefix as well.

Reply:
And IIUC, this test is for the other group of metrics mentioned above, so the appearance of `kubevirt_vmi_migration_failed` and `kubevirt_vmi_migration_succeeded` may cause a false positive.
Unless, those are not collected by this `TakeMetricsWithPrefix()`.

Reply:
So, in that case, the test would need to check from a specific list.

Reply:
Feel free to unhold upon resolution. From what I see the test is checking non-zero values in the collected metrics. So yes you will have to filter the metrics so the list will be the same as it was before your change

Reply:
In general I don't know whether it makes sense to maintain `test_id:8482` , maybe we can remove it? it looks like we test nothing but Prometheus functionality itself in the test, w/o reasoning about the metric values.

Reply:
I updated the test to look specifically for the relevant metrics. Please take a look and let me know if that's OK.